packages/main/src/plugin/kubernetes/contexts-states.ts

```diff
@@ -135,7 +201,83 @@ export class ContextsStates {
     update(val);
   }
 
-  async dispose(name: string): Promise<void> {
+  dispatch(options: DispatchOptions): void {
```

Comment:
Could we have a few line of documentation above, TBH I am not sure to understand what this is doing :'(